finder-app/makefile

```diff
@@ -0,0 +1,22 @@
+# Default compiler and compiler flags
+CC := gcc
+CF := -Wall
+
+# If Cross Compiler provided as argument
+ifdef CROSS_COMPILE
```

Comment:
@akka2103 see simplified suggestion for this logic in the assignment slides/video.

Reply:
Thankyou so much for the feedback.  Will definitely look into intended implementation.